pages/linux/clamscan.md

```diff
@@ -0,0 +1,21 @@
+# clamscan
+
+> Open-source anti-virus program.
+> Designed especially for e-mail scanning on mail gateways, but can be used in other contexts.
+> More information: <https://www.clamav.net>.
+
+- Update virus definitions:
+
+`freshclam`
```

Comment:
Hey @sbrl we have another command in this page. What we should do?

Reply:
We could link to another page for "freshclam" by adding a "See also: " section in the header.

Reply:
freshclam and clamscan are both part of the clamav, it would be better to leave clamav as the main page and freshclam nad clamscan as commands because they are not distinct tools and are usable only in this context, so they are just like clam flags but not implemented as such.

Reply:
Ah gotcha. I'm OK with that.